arrow-version/_code/core-home-code.md

```diff
@@ -1,7 +1,9 @@
 ---
 library: core
 ---
-```kotlin:ank:playground
+
+{: data-executable="true"}
+```kotlin
 import arrow.*
 import arrow.core.*
 
```

Comment:
Given that we're here and it's an easy change, could you change this snippet to call `magic` with a value?

Also, given that we're trying to convince people that there isn't any undecipherable magic involved, I'd rename that variable to `chain` or `combination`.

Reply:
Ok @pakoito, I will try it. Keep in mind that we are still not including the definitive code fragment for each library.